Source/ARTDefault+Private.h

```diff
@@ -9,7 +9,7 @@
 #import <Ably/ARTDefault.h>
 
 extern NSString *const ARTDefaultProduction;
-
+extern NSString *const ARTDefault_libName;
```

Comment:
Please avoid abbreviation. Perhaps rename to `ARTDefault_libraryName`.

Reply:
https://github.com/ably/ably-cocoa/pull/1119/commits/9eb087a3fb274c10ef78e479154d38278a027228